go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/mattn/go-isatty v0.0.16
 	github.com/mongodb-forks/digest v1.0.4
 	github.com/mongodb-labs/cobra2snooty v0.12.2
-	github.com/mongodb/mongodb-atlas-kubernetes v1.5.1
+	github.com/mongodb/mongodb-atlas-kubernetes v1.5.2-0.20230108093346-a8b6093d1e8d
```

Comment:
[q] when will v1.5.3 be released?

Reply:
The release is planned for early/mid-February. But I will ask the team if we can release the release sooner.

Reply:
Regarding this:

> mongodb-atlas-kubernetes dependency is not set to the release version, because the feature is not released yet. A proper version (1.6.0) will be set as soon as it will be released.

I think we may want to wait for 1.6.0 to be released before merging this PR. AtlasCLI must be always ready to be released at any time and we cannot release atlascli with this feature without the kub 1.6.0.